addons/hr_recruitment_event/report/hr_applicant_event_report.py

```diff
@@ -0,0 +1,69 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, tools
+
+
+class HrApplicantEventReport(models.Model):
+    _auto = False
+    _name = 'hr.applicant.event.report'
+    _description = 'Applicant Event Report'
+
+    active = fields.Boolean(readonly=True)
+    event_id = fields.Many2one('event.event', readonly=True)
+    event_name = fields.Char(readonly=True)
+    applicant_name = fields.Char(readonly=True)
+    event_country_id = fields.Many2one('res.country', readonly=True)
+    number_of_attendees = fields.Integer('Number of participants', readonly=True)
+    number_of_applicants = fields.Integer('Number of applicants', readonly=True)
+    number_of_signed_contracts = fields.Integer('Number of signed contracts', readonly=True)
+    date_begin = fields.Date(readonly=True)
+
+    def _select(self):
+        return """
+                event.active AS active,
+                e.event_id AS event_id,
+                event.name AS event_name,
+                a.name AS applicant_name,
+                date_begin AS date_begin,
+                event.country_id AS event_country_id,
+                COUNT(e.id) AS number_of_attendees,
+                COUNT(a.id) AS number_of_applicants,
+                COUNT(a.id) FILTER (
+                                    WHERE a.active
+                                          AND refuse_reason_id IS NULL
+                                          AND a.date_closed IS NOT NULL ) AS number_of_signed_contracts
```

Comment:
Like this with a parsed sql query like this the readability is pretty low already so I would suggest to find better aliases like applicant event and not giving alias to event_registration